lib/tupelo.js

```diff
@@ -371,19 +374,142 @@ class TupeloClient {
    * @property {string} tip - The chain tree tip after the transaction.
    */
 
-   /**
+  /**
+   * Build an establish token transaction object to use in a play transactions
+   * request
+   *
+   * @param {string} tokenName - Name of the new token
+   * @param {number} maximum - Maximum number of tokens of this type that can exist
+   *
+   * @return {EstablishTokenTransaction}
+   */
+  establishTokenTransaction(tokenName, maximum){
+    return {
+      type: "ESTABLISHTOKEN",
+      establishTokenPayload: {
+        name: tokenName,
+        monetaryPolicy: {
+          maximum: maximum,
+        }
+      }
+    };
+  }
+  /**
+   * @typedef {Object} EstablishTokenTransaction
+   * @property {string} name - Name of the new token
+   * @property {TokenMonetaryPolicy} monetaryPolicy - The tokens monetary policy
+   */
+
+  /**
+   * @typedef {Object} TokenMonetaryPolicy
+   * @property {number} maximum - Maximum total tokens allowed
+   */
+
+  /**
+   * Build a mint token transaction object to use in a play transactions request
+   *
+   * @param {string} tokenName - Name of the token type
+   * @param {number} amount - Number of tokens to mint.
+   *
+   * @return MintTokenTransaction
+   */
+  mintTokenTransaction(tokenName, amount){
+    return {
+      type: "MINTTOKEN",
+      mintTokenPayload: {
```

Comment:
same here

Reply:
yeah. this is part of the "quick and dirty" method because using the protobuf objects directly require generated code. the proto loader [converts objects with the correct field names and structure into protos](https://github.com/protobufjs/protobuf.js#usage) at runtime. i think we should use generated code loaded from a dependency eventually, but this should also work as a stopgap for now.

Reply:
I do use the proto objects in [the other branch](https://github.com/quorumcontrol/tupelo.js/compare/feature/play-transactions?expand=1), but there are still a lot of other issues to work out there.